scripts/actions/mobskills/fluorescence.lua

```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+-- Fluorescence
+-- Description: Emits radiation to boost next attack.
+-- Type: Enhancing
+-- Utsusemi/Blink absorb: N/A
+-- Range: Self
+-----------------------------------
+---@type TMobSkill
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.BOOST, 4, 0, 5))
```

Comment:
```suggestion
    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.BOOST, 400, 0, 5))
```
Jimmy's sheet says `"Boost" effect of * 4`. Boost gives ATTP as 1% per power, so 400% ATTP = 4x attack

I'm getting clarification from him to confirm he means 400% attack

Reply:
Jimmy confirmed it's 400% attack

Reply:
Well then, I way underestimated that. I was assuming it meant a 4% increase. Got that adjusted.

Reply:
> Well then, I way underestimated that. I was assuming it meant a 4% increase. Got that adjusted.

Right. that's why I asked him haha